### PR TITLE
Add release notes for 1.0-beta release

### LINKED ISSUE #765: We should let the user know 'what's new' when they upgrade our plugin
I believe the plugin description is displayed during the update process. We should verify that and possibly merge our CHANGELOG.md into the descriptor at release.

### PR DESCRIPTION
**Pull request by aslo**
fixes #765

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [1.0-beta] - 2016-09-14
+### Added
+- App Engine Standard support ([#767](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/767))
```
Comment (etanshaul) [inline CHANGELOG.md]: I think the way they want us calling this is: 'App Engine standard environment' support

File: RELEASING.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 This page describes how to cut releases of our plugin. It's strictly for the owners of the project.
 
+1. Update [CHANGELOG.md](https://github.com/GoogleCloudPlatform/google-cloud-intellij/blob/master/CHANGELOG.md) with any major release notes (see [guidelines](http://keepachangelog.com/en/0.3.0/)). Replace the contents of the `<change-notes>` tag in [plugin.xml](https://github.com/GoogleCloudPlatform/google-cloud-intellij/blob/master/google-cloud-tools-plugin/resources/META-INF/plugin.xml) with the same notes, in html.
```
Comment (etanshaul) [inline RELEASING.md]: Maybe a link to a markdown compiler? If you think that's inappropriate maybe change "in html" to .."with the same notes compiled from markdown to html".

File: CHANGELOG.md (hunk 1 above)
Comment (etanshaul) [inline CHANGELOG.md]: A couple other changes that would be nice to include in the log:

https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/868 (exposing extra fields in the deployment config)
https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/811 (updating local run debug to use the newer tooling)

Comment (aslo) [inline CHANGELOG.md]: sounds good!

File: RELEASING.md (hunk 2 above)
Comment (aslo) [inline RELEASING.md]: I like "with the same notes compiled from markdown to html". Will add that.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,10 +3,18 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [1.0-beta] - 2016-09-14
+### Added
+- App Engine standard environment support ([#767](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/767))
+
+### Changed
+- Extra fields now available in the deployment config ([#868](https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/868))
```
Comment (etanshaul) [inline CHANGELOG.md]: Maybe put this under the added section.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,10 +3,18 @@ All notable changes to this project will be documented in this file.
...
+- Local run debug updated to use the newer tooling ([#811](https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/811))
```
Comment (etanshaul) [inline CHANGELOG.md]: on second thought this is an internal implementation detail. maybe not needed in the changelog

**Comment by etanshaul** [comment]
lgtm after my last couple comments

**Pull request merged**